### ISSUE TITLE
Fatal error and warnings after last git pull

### ISSUE THREAD

**Issue by AlfredSK** [friendica/friendica#6334]
Three issues found in PHP log after PR #6333 (probably related to a previous PR).

- [ ] fatal error in DBA.php
````
PHP Fatal error:  Uncaught Error: Call to undefined function Friendica\Database\get_app() in /var/www/html/libranet/src/Database/DBA.php:391
Stack trace:
#0 /var/www/html/libranet/src/Database/DBA.php(285): Friendica\Database\DBA::p('SELECT 1')
#1 /var/www/html/libranet/src/App/Mode.php(63): Friendica\Database\DBA::connected()
#2 /var/www/html/libranet/src/App.php(306): Friendica\App\Mode->determine('/var/www/html/l...')
#3 /var/www/html/libranet/src/App.php(184): Friendica\App->reload()
#4 /var/www/html/libranet/index.php(11): Friendica\App->__construct('/var/www/html/l...', false)
#5 {main}
  thrown in /var/www/html/libranet/src/Database/DBA.php on line 391
````

- [ ] warning in settings.config.php
````
PHP Warning:  Use of undefined constant REGISTER_CLOSED - assumed 'REGISTER_CLOSED' (this will throw an Error in a
future version of PHP) in /var/www/html/libranet/config/settings.config.php on line 22
````

- [ ] warning in items.php
````
PHP Warning:  preg_replace(): Compilation failed: missing ) at offset 82 in /var/www/html/libranet/include/items.php on line
207
````

### Friendica version you encountered the problem
2018.12-rc 

### Friendica source (git, zip)
git

### PHP version
7.2.10

### SQL version
10.1.34 MariaDB

**Comment by annando** [comment]
Can you reproduce the error? Possibly it occurred during the update?

**Comment by AlfredSK** [comment]
Hmm. I was thinking about that too. I'll keep the issue open and will have a look at the log tomorrow morning.

**Comment by MrPetovan** [comment]
I'm getting the error too, a fix in on the way.

**Comment by AlfredSK** [comment]
I checked my PHP logs again. The issues didn't reappear in the last 15 hours. :thinking:

**Comment by MrPetovan** [comment]
Closed by #6372
